butterknife-compiler/src/main/java/butterknife/compiler/BindingClass.java

```diff
@@ -39,31 +42,40 @@
   private static final ClassName CONTEXT = ClassName.get("android.content", "Context");
   private static final ClassName RESOURCES = ClassName.get("android.content.res", "Resources");
   private static final ClassName THEME = RESOURCES.nestedClass("Theme");
+  private static final ClassName BUTTERKNIFE = ClassName.get("butterknife", "ButterKnife");
   private static final ClassName UNBINDER =
```

Comment:
This can be `BUTTERKNIFE.nestedClass("ViewUnbinder")` to be in-line with `RESOURCES` and `THEME`